Sources/Apodini/Semantic Model Builder/SharedSemanticModelBuilder.swift

```diff
@@ -94,10 +94,10 @@ class SharedSemanticModelBuilder: SemanticModelBuilder, InterfaceExporterVisitor
         
         webService.finish()
         
-        webService.root.printTree() // currently only for debugging purposes
+        app.logger.info(.init(stringLiteral: webService.root.debugDescription))
```

Comment:
Is the init needed? Shouldn't LogMessage be able to be implicitly constructed from a String instance?

Reply:
Yes, it's needed 😕